doc/build-instructions/Build_Instructions_V13.md

```diff
@@ -0,0 +1,654 @@
+
+:blue_book: *Congratulations!* :tada:
+
+----------------------------------
+
+## Windows
+:ledger:
+
+The following instructions guide you through the process of building a Windows **OpenJDK V13** binary that contains Eclipse OpenJ9. This process can be used to build binaries for Windows 7, 8, and 10.
+
+### 1. Prepare your system
+:ledger:
+You must install a number of software dependencies to create a suitable build environment on your system:
+
+- [Cygwin](https://cygwin.com/install.html), which provides a Unix-style command line interface. Install all packages in the `Devel` category. Included in this package is mingw, a minimalist subset of GNU tools for Microsoft Windows. OpenJ9 uses the mingw/GCC compiler and is tested with version 6.4.0 of same. Other versions are expected to work also. In the `Archive` category, install the packages `zip` and `unzip`. In the `Utils` category, install the `cpio` package. Install any further package dependencies that are identified by the installer. More information about using Cygwin can be found [here](https://cygwin.com/docs.html).
+- [Windows JDK 10](https://adoptopenjdk.net/releases.html?variant=openjdk11#x64_win), which is used as the boot JDK.
+- [Microsoft Visual Studio 2017]( https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=15), which is the default compiler level used by OpenJDK13; or [Microsoft Visual Studio 2013]( https://go.microsoft.com/fwlink/?LinkId=532495), which is chosen to compile if VS2017 isn't installed.
+- [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download)
+- [LLVM/Clang](http://releases.llvm.org/7.0.0/LLVM-7.0.0-win64.exe)
+- [NASM Assembler v2.13.03 or newer](https://www.nasm.us/pub/nasm/releasebuilds/?C=M;O=D)
+
+Add the binary path of Clang to the `PATH` environment variable to override the older version of clang integrated in Cygwin. e.g.
+```
+export PATH="/cygdrive/c/LLVM/bin:$PATH" (in Cygwin)
+```
+
+Add the path to `nasm.exe` to the `PATH` environment variable to override the older version of NASM installed in Cygwin. e.g.
+```
+export PATH="/cygdrive/c/NASM:$PATH" (in Cygwin)
+```
+
+Update your `LIB` and `INCLUDE` environment variables to provide a path to the Windows debugging tools with the following commands:
+
+```
+set INCLUDE=C:\Program Files\Debugging Tools for Windows (x64)\sdk\inc;%INCLUDE%
+set LIB=C:\Program Files\Debugging Tools for Windows (x64)\sdk\lib;%LIB%
+```
+
+   You can download Visual Studio, Freemarker manually or obtain them using the [wget](http://www.gnu.org/software/wget/faq.html#download) utility. If you choose to use `wget`, follow these steps:
+
+- Open a cygwin terminal and change to the `/temp` directory:
+```
+cd /cygdrive/c/temp
+```
+
+- Run the following commands:
+```
+wget https://go.microsoft.com/fwlink/?LinkId=532495 -O vs2013.exe
```

Comment:
should be 2017 for jdk11 and above. I expect this means the link also needs updating

Reply:
I can't find what that link should be.